src/hb-face.cc

```diff
@@ -623,20 +624,7 @@ hb_face_collect_variation_unicodes (hb_face_t *face,
 
 struct hb_face_builder_data_t
 {
-  struct table_entry_t
-  {
-    int cmp (hb_tag_t t) const
-    {
-      if (t < tag) return -1;
-      if (t > tag) return -1;
-      return 0;
-    }
-
-    hb_tag_t   tag;
-    hb_blob_t *blob;
-  };
-
-  hb_vector_t<table_entry_t> tables;
+  hb_hashmap_t<hb_tag_t, hb_blob_t*, (unsigned)-1, nullptr> tables;
```

Comment:
The template defaults for the last two args should be enough.

Reply:
I tried this but unfortunately it seems to choke finding the default for the hb_blob_t* pointer.

Reply:
Okay fixed it.